specification/index.bs

```diff
@@ -190,7 +199,70 @@ Issue(62): Specify localization handling.
 
 ### Content scripts
 
-#### Isolated worlds
+<dfn>Content scripts</dfn> represent a set of JS and CSS files that should be injected into pages loaded by the user agent.
+
+#### Key `matches`
+
+A [=list=] of [=match patterns=] that are used to decide where the content script runs. This key is required.
```

Comment:
```suggestion
A [=list=] of [=match patterns=] that are used to which pages the user agent injects the content script into. This key is required.
```

Reply:
Devlin's suggestion appears to have a dropped a word.

```suggestion
A [=list=] of [=match patterns=] that are used to decide which pages the user agent injects the content script into. This key is required.
```